moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -212,35 +204,40 @@ bool Servo::validateParams(const servo::Params& servo_params)
 
 bool Servo::updateParams()
 {
```

Comment:
You ought to check `getStatus() != StatusCode::PAUSED` right away and return immediately if it is. Reason being, `validateParams()` is expensive and you wouldn't want somebody calling it from within a control loop.

Then you don't need to check the status code at L219,L225.

```suggestion
{
  if (getStatus() != StatusCode::PAUSED)
  {
    RCLCPP_WARN_STREAM_THROTTLE(LOGGER, *node_->get_clock(), ROS_LOG_THROTTLE_PERIOD,
                                  "Updating parameters without pausing Servo is not allowed");
  }
```

Reply:
Yeah :+1: 
Will include this in the upcoming PR since that PR has some other changes that are need for planning group anyway.